modules/custom/d_demo/pages/contact.yml

```diff
@@ -61,7 +65,7 @@ field_page_section:
           data: 'Our offices'
         field_d_settings:
           type: field_p_configuration_storage
-          custom_class: 'full-width with-tiles background-gray-light-2'
+          custom_class: full-width with-tiles background-gray-light-2 theme-default padding-top-default padding-bottom-default margin-top-default margin-bottom-default
```

Comment:
tutaj to samo co w przypadku `.theme-invert` `.background-gray-light-2` ma byc zamieniony docelowo na `.theme-gray`, wiec ten default tutaj troche nie pasuje